src/kit/Progress.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import { floatToPercent } from 'kit/internal/string';
+import { ShirtSize } from 'kit/Theme';
+import Tooltip from 'kit/Tooltip';
+
+import css from './Progress.module.scss';
+
+export interface BarPart {
+  bordered?: string;
+  color: string; // css color
+  label?: string;
+  percent: number; // between 0-1
+}
+
+export interface Props {
+  inline?: boolean;
+  parts: BarPart[];
+  showLegend?: boolean;
+  size?: ShirtSize;
+  title?: string;
+}
+
+const partStyle = (part: BarPart) => {
+  let style = {
+    backgroundColor: part.color,
+    borderColor: 'var(--theme-float-border)',
+    borderStyle: 'none',
+    borderWidth: 1,
```

Comment:
do we need to define these here instead of in the scss?

Reply:
✅  I can move these lines into the scss and leave just the parts derived from `part.color` and `part.percent`